coffee_httpd/src/httpd/server.rs

```diff
@@ -64,6 +68,39 @@ async fn coffee_help(
     Ok(body)
 }
 
+#[api_v2_operation]
+#[get("/list")]
+async fn coffee_list(
+    data: web::Data<AppState>,
+) -> Result<Json<HashMap<String, Vec<HashMap<String, String>>>>, Error> {
+    let mut coffee = data.coffee.lock().await;
+    let result = coffee.list().await;
```

Comment:
```suggestion
    let result = coffee.list().await;
```

Why the coffee list is not returning directly a model that we can you inside the JSON response? as 

```rust
json!({ "plugin": result })
```

Reply:
We can but there will be additional information shown to the user.

Reply:
The Rest response and the core response should be the same, so what additional response are you returning to the user?